tx.go

```diff
@@ -571,8 +600,9 @@ func (tx *Tx) rollbackChanges() error {
 
 	sz, err := tx.file.file.Size()
 	if err != nil {
-		// getting file size failed. State is valid, but we can not truncate :/
-		return err
+		// getting file size failed. State is valid, but we can not truncate
+		// ¯\_(ツ)_/¯
+		return
```

Comment:
Still sounds like an error to me worth noting?

Reply:
It's not really an error in this location. The file and txfile state is still valid, but the OS failed us when returning bytes to the filesystem. This does not affect txfile from operating correctly. Next transaction will try to truncate again, until we succeed at some point in time.